README.md

```diff
@@ -1,5 +1,7 @@
 # Europeana::Logging
 
+[![Build Status](https://travis-ci.org/europeana/radio-station.svg?branch=develop)](https://travis-ci.org/europeana/radio-station) [![Security](https://hakiri.io/github/europeana/radio-station/develop.svg)](https://hakiri.io/github/europeana/radio-station/develop) [![Maintainability](https://api.codeclimate.com/v1/badges/b730a787cbe77e612784/maintainability)](https://codeclimate.com/github/europeana/radio-station/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/b730a787cbe77e612784/test_coverage)](https://codeclimate.com/github/europeana/radio-station/test_coverage)
```

Comment:
Those badge IDs and/or link URLs are for the radio station repo. For this gem the badges are:
```markdown
[![Maintainability](https://api.codeclimate.com/v1/badges/d96137b6a5ccb853209d/maintainability)](https://codeclimate.com/github/europeana/europeana-logging-ruby/maintainability)
[![Test Coverage](https://api.codeclimate.com/v1/badges/d96137b6a5ccb853209d/test_coverage)](https://codeclimate.com/github/europeana/europeana-logging-ruby/test_coverage)
```